Classes/ToxWrapper/OCTTox.m

```diff
@@ -711,6 +715,34 @@ - (int)fileSendRequestWithFriendNumber:(int32_t)friendNumber fileName:(NSString
     return result;
 }
 
+- (int)fileSendControlWithFriendNumber:(int32_t)friendNumber sendReceive:(uint8_t)sendReceive fileNumber:(uint8_t)fileNumber controlType:(uint8_t)controlType data:(const uint8_t *)data length:(uint16_t)length
+{
+    int result = tox_file_send_control(self.tox, friendNumber, sendReceive, fileNumber, controlType, data, length);
```

Comment:
It would be better to match our controlType from enum with enum from toxcore. You cannot be sure that after update in the future those types will match. Something like

```objc
switch (contolType) {
    case OCTToxFileControlAccept:
        cControlType = TOX_FILECONTROL_ACCEPT;
        ...
};